lib/modules/commentNavigator.js

```diff
@@ -89,7 +88,7 @@ modules['commentNavigator'] = {
 						case 'images':
 							thisEle.setAttribute('title', 'Navigate through comments with images');
 							break;
-						case 'images':
+						case 'videos':
```

Comment:
the symlinks work such that the extension loads just fine, it's just that edits with IntelliJ don't seem to propagate to the linked files, whereas edits with SublimeText do.

using gulp and copying files over each time sucks... requires a "watch" command which is lame, but I guess if I worked something into my workflow to automate starting that on save, maybe it wouldn't be such a PITA.